### PR TITLE
QDB-3131 - Symbols .NET API

### PR DESCRIPTION
**Pull request by Dwarfobserver**
Adds symbols, use create/list_columns/insert_columns _ex.
It seems some symbol keys are refused (when generated from random bytes).

### CODE REVIEW

File: Quasardb.Tests/Cluster/Writer.cs [hunk 1]
```diff
@@ -96,20 +98,39 @@ public QdbTimestampPointCollection CreateTimestampPoints(DateTime time, int coun
...
 
+        public QdbSymbolPointCollection CreateSymbolPoints(DateTime time, int count)
+        {
+            Random random = new Random();
+            var r = new QdbSymbolPointCollection(count);
+
+            for (int i = 0; i < count; ++i)
+            {
+                /*var value = new byte[32];
+                random.NextBytes(value);
```
Comment (mkurdej) [inline Quasardb.Tests/Cluster/Writer.cs:109]: That's related with https://app.clubhouse.io/quasardb/story/9299/symbol-api-refuse-some-keys-from-random-bytes?
You just shouldn't use random bytes but a random string generator. Just narrow down to UTF-8 or even alphanumeric.
E.g. https://stackoverflow.com/questions/1344221/how-can-i-generate-random-alphanumeric-strings.

Comment (Dwarfobserver) [reply Quasardb.Tests/Cluster/Writer.cs:109]: Yes I created the ticket after this, I'll use the SO generator thank you

**Review by mkurdej — APPROVED** [review]
LGTM. Please add a note in the commit message about the UTF16/8 fix.

**Comment by Dwarfobserver** [comment]
Supports symbols.
/!\ Strings (and symbols) are now stored in QuasarDB as UTF-8 strings (they were UTF-16 strings before).

**Pull request merged**